### PR TITLE
Adding Snapshot and Version actions to LCM

### PR DESCRIPTION
**Pull request by HimanshuChhabra**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json [hunk 1]
```diff
@@ -3242,6 +3245,10 @@
         "snapshot": {
           "$ref": "#/definitions/ManagementPolicySnapShot",
           "description": "The management policy action for snapshot"
+        },
+        "version": {
```
Comment (Juliehzl) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json:3249]: May I know why ManagementPolicyAction is required in a PolicyDefinition, but no one property in ManagementPolicyAction is required? 
So I can create a policy with empty action info, right?

Comment (HimanshuChhabra) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json:3249]: Yes, the properties within the actions are optional.

**Review by Juliehzl — APPROVED** [review]

**Comment by ravbhatnagar** [comment]
Adding new properties to an existing api-version is not allowed as per the Azure REST API versioning guidelines. Please work with Azure API review board to get this approved. @adrianhall @johanste

**Comment by Juliehzl** [comment]
Hi @HimanshuChhabra, do we still need the PR? as I know, storage team created a new Preview version for new properties

**Comment by Juliehzl** [comment]
Hi @HimanshuChhabra, given there is no response for a long time, I have to close the PR and when it is ready, feel free to ask me to reopen it.
But you may need to pay attention to the breaking change policy as shown above.

**Pull request closed without merging**